skapa-ett-konfetti-regn/confetti.js

```diff
@@ -0,0 +1,49 @@
+// Om du tittar på projektet i browsern ser du en ensam grå
+// och lite sorglig konfettipartikel. Din uppgift blir att
+// fylla konfettianimationen med färger och liv!
+
+// Funktionen chooseParticleColor styr färgen på varje enskild
+// partikel. Den ska returnera en text-sträng som innehåller en
+// CSS-färg. Returnerar den samma färg varje gång så kommer alla
+// partiklar ha samma färg men om den returnerar olika färger
+// t.ex. baserat på slump eller baserat på inparametern groupName
+// så kan det istället bli en blandning av färger.
+const chooseParticleColor = (groupName) => {
+    let letters = '0123456789ABCDEF'; //Symbolerna som kan användas i en hexcode
+    let color = '#'; //En hexcode måste börjas med # 
+    for (let i = 0; i < 6; i++) { 
+      color += letters[Math.floor(Math.random() * 16)]; //Loopar igenom letters och väljer 6 slumpmässiga symboler. kör sedan # + symbolerna för att bilda en hexcode
+    }
+  return color; //Variabeln color returneras, som är en slumpmässig hexcode (slumpmässig färg)
```

Comment:
Den här lösningen funkar absolut, den löser uppgiften och slumpar in alla möjliga färger som finns, snyggt!

Tips för den här typ av lösning är att man skulle kunna välja att returnera RGB värden och jobba direkt med siffror, som i RGB(0, 0, 0);

Till exempel
```
return `RGB(${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)})`
```
Eller för hex värden så har Nummer i javascript en toString metod där man kan skicka in 16 för att konvertera till hex. Så Number(255).toString(16) ger 'f'. Då kan man även returnera en del färgkod baserad på ett slumpat värde mellan 0 och 16777215 som är max antal olika färger man kan få från RGB 0, 0, 0 till RGB 255, 255, 255.

Det kan då se såhär ut
```
return '#' + Math.floor(Math.random()*16777215).toString(16)
```